code/modules/vampire_neu/covens/coven_powers/quietus.dm

```diff
@@ -0,0 +1,147 @@
+/datum/coven/quietus
+	name = "Quietus"
+	desc = "Get a help from the Hell creatures, resist THE FIRE, transform into an imp. Violates Masquerade."
+	icon_state = "daimonion"
+	power_type = /datum/coven_power/quietus
+	clan_restricted = TRUE
+
+/datum/coven_power/quietus
+	name = "Quietus power name"
+	desc = "Quietus power description"
+
+//SILENCE OF DEATH
+/datum/coven_power/quietus/silence_of_death
+	name = "Silence of Death"
+	desc = "Create an area of pure silence around you, confusing those within it."
+
+	level = 1
+	research_cost = 0
+	check_flags = COVEN_CHECK_CAPABLE | COVEN_CHECK_CONSCIOUS | COVEN_CHECK_IMMOBILE | COVEN_CHECK_LYING
+	duration_length = 5 SECONDS
+	cooldown_length = 15 SECONDS
+	duration_override = TRUE
+
+/datum/coven_power/quietus/silence_of_death/activate()
+	. = ..()
+	for(var/mob/living/carbon/human/H in get_hearers_in_range(7, owner))
```

Comment:
I've redone this on my branch since I'm adding these to vanderlin, it really should be a toggle and use a proximity field